shared/redux/rtk-apis/assignments/assignments.api.ts

```diff
@@ -58,6 +58,24 @@ const assignmentsApi = projectApi.injectEndpoints({
       }),
       invalidatesTags: (_, __, { classroomId }) => [{ type: "Assignments", id: classroomId }],
     }),
+
+    submitAssignment: builder.mutation<
+      void,
+      { assignmentId: number; classroomId: number; userId: number; file: File }
+    >({
+      query: ({ classroomId, assignmentId, userId, file }) => {
+        const formData = new FormData();
+        formData.append("file", file);
+        formData.append("assignmentId", assignmentId.toString());
+        formData.append("userId", userId.toString());
+        return {
+          url: `classrooms/${classroomId}/assignments/${assignmentId}/submit-assignment`,
+          method: "POST",
+          body: formData,
+        };
+      },
+      invalidatesTags: (_, __, { classroomId }) => [{ type: "Assignments", id: classroomId }],
```

Comment:
I am unsure whether this should be inside the `assignments.api.ts` or in a separate file `submissions.api.ts` since this would add a new submission record in the backend.

@ragib-hossain-sazim bhai, what do you think?

Reply:
It makes sense to put it inside a seperate file `submissions.api.ts` because if we want to perform edit or delete submission actions we would need to add more api calls here.